### PR TITLE
PICARD-1216, 1217: Enable console mode to display STDOUT 

### PR DESCRIPTION
**Pull request by samj1912**
# Summary

* This is a…
    * [x] Bug fix
    * [ ] Feature addition
    * [ ] Refactoring
    * [ ] Minor / simple change (like a typo)
    * [ ] Other
* **Describe this change in 1-2 sentences**:

# Problem

* JIRA ticket (_optional_): [PICARD-1216](https://tickets.metabrainz.org/browse/PICARD-1216)

# Solution

# Action

### CODE REVIEW

**Comment by sambhav** [comment]
Was a test PR. Doesn't seem to be working. :\

**Comment by sambhav** [comment]
So the console flag does show the output, but adds an additional console that pops up each time you open Picard. I'd say we close this PR and investigate how to properly setup stdout on Windows, since it doesn't seem to be affecting osx or linux.

File: picard/util/__init__.py [hunk 1]
```diff
@@ -39,6 +39,20 @@
 is_frozen = getattr(sys, 'frozen', False)
 frozen_temp_path = getattr(sys, '_MEIPASS', '')
 
+def hide_console():
+    if platform.system() == 'Windows':
```
Comment (zas) [inline picard/util/__init__.py]: `sys.platform` is used elsewhere, and `platform` isn't imported (does it work?)

File: tagger.py [hunk 2]
```diff
@@ -6,12 +6,14 @@
...
 from picard.tagger import main
-from picard.util import is_frozen, frozen_temp_path
+from picard.util import is_frozen, frozen_temp_path, hide_console
 
 
 # This is needed to find resources when using pyinstaller
 if is_frozen:
     basedir = frozen_temp_path
+    # Hide console window on Windows
+    hide_console()
```
Comment (zas) [inline tagger.py]: Is the console window shown only when started from command line with this ?

**Comment by samj1912** [comment]
Inactive PR and doesn't seem to work. Closing for now.

**Pull request closed without merging**